src/card/card.jsx

```diff
@@ -21,6 +21,13 @@ const Card = React.createClass({
     expandable: React.PropTypes.bool,
 
     /**
+     * Whether this card is expanded.
+     * If `true` or `false` the component is controlled.
+     * if `null` the component is uncontrolled.
+     */
+    expanded: React.PropTypes.bool,
```

Comment:
`Is this dialog open?` vs. `Is this card expand?` 

Open can be an adjective but not expand, it's only a verb. I think we should stick to `expanded`. these are properties after all. so they should be adjectives I believe. but @mbrookes can provide better insight than me O.o